src/utility/Utility.h

```diff
@@ -17,29 +17,25 @@
 #include "spdlog/sinks/stdout_color_sinks.h"
 #endif
 
-class Utility {
- public:
-    static std::vector<size_t> coordinateFromLinearIndex(size_t idx,
+namespace Utility {
+    std::vector<size_t> coordinateFromLinearIndex(size_t idx,
                                                          size_t Nx,
                                                          size_t Ny);
-//  static size_t getCoordinateI(size_t idx,
-//                               size_t Nx,
-//                               size_t Ny,
-//                               size_t j,
-//                               size_t k);
-//  static size_t getCoordinateJ(size_t idx, size_t Nx, size_t Ny, size_t k);
-//  static size_t getCoordinateK(size_t idx, size_t Nx, size_t Ny);
-    static std::vector<std::string> split(const char* text, char delimiter);
-    static std::vector<std::string> split(const std::string& text,
+//  size_t getCoordinateI(size_t idx,
```

Comment:
just remove the now unnecessary methods getCoordinateI/J/K

Reply:
done in a47401d